src/model/class.ts

```diff
@@ -151,7 +151,7 @@ export class Class implements Feature {
 
     this.warnings =
         init.warnings === undefined ? [] : Array.from(init.warnings);
-    this.demos = init.demos || jsdocLib.extractDemos(init.jsdoc, document.url);
+    this.demos = [...init.demos || [], ...jsdocLib.extractDemos(init.jsdoc)];
```

Comment:
If `init.demos` is always the empty array, we could consider making it optional and leaving it out most places?